doc/devel/contribute.rst

```diff
@@ -279,11 +112,26 @@ though not necessarily all at the same time:
 - It involves Python features such as decorators and context managers, which
   have subtleties due to our implementation decisions.
 
+.. _first_contribution:
+
+First Contributions & New Contributors
+--------------------------------------
+
+If this is your first open source contribution, or first time contributing to Matplotlib,
+and you need help or guidance finding a good first issue, look no further. This section will
+guide you through each step.
+
+1. Navigate to the `issues page <https://github.com/matplotlib/matplotlib/issues/>`_.
+2. Filter labels with `"Difficulty: Easy" <https://github.com/matplotlib/matplotlib/labels/Difficulty%3A%20Easy>`_ & `"Good first Issue" <https://github.com/matplotlib/matplotlib/labels/good%20first%20issue>`_ (optional).
+3. Click on an issue you would like to work on, and check to see if the issue is already claimed.
+4. If the issue is claimed, tag the user working on the issue, and ask to collaborate (optional).
+5. Check the `pull requests (i.e. #28476) <https://github.com/matplotlib/matplotlib/pulls?q=is%3Apr+is%3Aopen+%2328476>`_ and filter by the issue number to make sure the issue is not in progress.
+6. After steps 3 & 4 are done, and no one is working on the issue, comment on the issue to state you are working on it.
```

Comment:
This would be an organizational decision, but maybe set max time of no activity: no active comments/discussions within issue or idle/open PR, people can just work on the issue. This way you dont need to moderate, simple yes/no check. If inactive >1 month, free to take, else, try to contact the original contributor. 

Let me know what you think, because I think you're suggesting we just drop that entire line and rewrite it to include drafts?

Reply:
Thank you for sharing the link, unfortunately I am working during the meeting and cannot attend. I also created a new PR based on your recommendations and suggestions, thank you for the guidance!